### ISSUE TITLE
Grabber-DIOT 

### ISSUE THREAD

**Issue by gkasprow** [sinara-hw/Grabber#11]
- change description, we have 2024
- unmask the ESD strip

**Comment by jordens** [comment]
Sidenote: Looking at the DIOT modules, can't we already adopt some of the niceties from DIOT as design rules for EEM as well? Right now, I can see the chamfered PCB corners on the rear as being useful.

**Comment by gkasprow** [comment]
Right, I think it's time to make a design template and set of rules.
For the moment, my team is over 40 ppl at the WUT, and many of them are using DI/OT for various projects.

**Comment by gkasprow** [comment]
https://github.com/sinara-hw/DIOT-Template/issues/1
